lms/templates/courseware/accordion.html

```diff
@@ -6,73 +6,44 @@
 %>
 
 <%def name="make_chapter(chapter)">
-  <div class="chapter">
-      <%
-          if chapter.get('active'):
-              aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
-              active_class = ' class="active"'
-          else:
-              aria_label = chapter['display_name']
-              active_class = ''
-      %>
-      <h3 ${active_class} aria-label="${aria_label}">
-        <a href="#">
-          ${chapter['display_name']}
-        </a>
-      </h3>
-
-    <ul>
-      % for section in chapter['sections']:
-          <li class="${'active' if 'active' in section and section['active'] else ''} ${'graded'  if 'graded' in section and section['graded'] else ''}">
+<%
+if chapter.get('active'):
+    aria_label = _('{chapter}, current chapter').format(chapter=chapter['display_name'])
+    active_class = 'active"'
```

Comment:
This is actually okay. It looks different in the actual file and would throw JS errors otherwise.